File: Java/TravelNetworking/src/main/java/chat/network/FlightDestDepDTO.java
package chat.network;

import java.io.Serializable;
import java.time.LocalDate;
import java.time.LocalDateTime;

public class FlightDestDepDTO implements Serializable {
    private String destination;
    private LocalDate departure;

    public FlightDestDepDTO(String destination, LocalDate departure) {
        this.destination = destination;
        this.departure = departure;
    }

    public String getDestination() {
        return destination;
    }

    public void setDestination(String destination) {
        this.destination = destination;
    }

    public LocalDate getDeparture() {
        return departure;
    }

    public void setDeparture(LocalDate departure) {
        this.departure = departure;
    }

    @Override
    public String toString() {
        return "FlightDestDepDTO{" +
                "destination='" + destination + '\'' +
                ", departure=" + departure +
                '}';
    }
}

File: ProiectCS/travelNetworking/FlightDestDepDTO.cs
using System;

namespace travelNetworking
{

    [Serializable]
    public class FlightDestDepDTO
    {
        public string Destination { get; set; }
        public DateTime Departure { get; set; }

        public FlightDestDepDTO(string destination, DateTime departure)
        {
            Destination = destination;
            Departure = departure;
        }

        public override string ToString()
        {
            return Destination + ";" + Departure;
        }
    }
}